frontend/javascripts/admin/voxelytics/workflow_list_view.tsx

```diff
@@ -181,6 +181,25 @@ export default function WorkflowListView() {
             width: 200,
             render: renderProgress,
           },
+          {
+            title: "File Size",
+            key: "fileSize",
+            width: 200,
+            render: (run: RenderRunInfo) => (
+              <Tooltip
+                overlay={
+                  <>
+                    {formatBytes(run.taskCounts.fileSize)} •{" "}
```

Comment:
Is there a reason for the • char? I would have expected | or comma or dash.

Also, maybe add a disclaimer like “Note: manual changes on disk are not respected here“?

Reply:
Also, I think on other places (like organization storage report) we went for powers-of-ten filesize formatting. Are there still other places where powers-of-two is used? Should we unify them? I am in favor of powers-of-ten everywhere (but showing the correct unit is of course the most important, which is already ok here).

Reply:
Thanks for considering my feedback :) What do you think about my idea of a disclaimer concerning the manual changes on disk?